"error.cshtml" false error

Another one for the hardcoded classifications:
Error.cshtml is a view in my project and lines with this filename are displayed 'Error', should be 'Build text'.

I wonder if the regex should be smart enough to catch more file types, fex. Error.cs
The filename will always follow a path.

Here is another similar example. These show in red in the build output:

```
D:\ONOSYS\ONOSYSv4\12\21.RTPConversion\Dev\ONOSYS.Api\Services\DatabaseExceptionLogger.cs(142,21,142,33): warning CS1591: Missing XML comment for publicly visible type or member 'DatabaseExceptionLogger.LogException(Exception)'

D:\ONOSYS\ONOSYSv4\12\21.RTPConversion\Dev\ONOSYS.Api\Services\DatabaseExceptionLogger.cs(163,21,163,33): warning CS1591: Missing XML comment for publicly visible type or member 'DatabaseExceptionLogger.LogException(Exception, string)'```

Whereas this one is fine (shows in the dark yellow color):
```
D:\ONOSYS\ONOSYSv4\12\21.RTPConversion\Dev\ONOSYS.Api\Services\DatabaseExceptionLogger.cs(148,30,148,45): warning CS1591: Missing XML comment for publicly visible type or member 'DatabaseExceptionLogger.GetDatabaseName()'```

A solution I sue for removing the false positives is below
eg:
Sample "error" highlighted line is in red but is only a warning.
ReviewRepo.cs(131,28): warning CS0168: The variable 'exception' is declared but never used

The following works for me, ie. check if it is NOT preceded with a warning an BC or CS reference number directly after the warning. I'm not sure it if needs cater for any other warning number prefixes (BC or CS etc)

MyRegExPattern[4] beomes:
(\W|^)^(?!.*warning\s(BC|CS)\d+:).*(error|fail|failed|exception)[^\w\.]

This should fix the previous comment from matthewpavkov as well

--- a/Tests/OutputClassifierTests.cs
+++ b/Tests/OutputClassifierTests.cs
@@ -54,6 +54,7 @@ namespace Tests
         [TestCase(" 0 Warning(s)", ClassificationTypeDefinitions.BuildHead)]
         [TestCase(" 0 Error(s)", ClassificationTypeDefinitions.BuildHead)]
         [TestCase("Could not find file", ClassificationTypeDefinitions.LogError)]
+        [TestCase("warning CS0168: The variable \'exception\'", ClassificationTypeDefinitions.LogWarn)]
         public void GetClassificationSpansFromSnapShot(string pattern, string classification)
         {
             Settings.Load();

--- a/VSColorOutput/State/Settings.cs
+++ b/VSColorOutput/State/Settings.cs
@@ -167,7 +167,7 @@ namespace VSColorOutput.State
                 },
                 new RegExClassification
                 {
-                    RegExPattern = @"(\W|^)(error|fail|failed|exception)[^\w\.]",
+                    RegExPattern = @"(\W|^)^(?!.*warning\s(BC|CS)\d+:).*(error|fail|failed|exception)[^\w\.]",
                     ClassificationType = ClassificationTypes.LogError,
                     IgnoreCase = true
                 },

